Dockerfile

```diff
@@ -18,7 +18,7 @@ ENV GODEBUG="netdns=go http2server=0"
 
 RUN make build BUILD_VERSION=${BUILD_VERSION}
 
-FROM alpine:3.13.4
+FROM alpine:latest
```

Comment:
This is Bad™, specially considering the rest of the dependencies are not locked into a specific version. Many things can go wrong in the long run that we can't really foresee. Best to introduce maintenance cycles with specific update exercises to ensure infrastructure quality.